Client/package.json

```diff
@@ -12,9 +12,6 @@
     "@material-ui/icons": "^4.9.1",
     "@material-ui/lab": "^4.0.0-alpha.48",
     "@material-ui/pickers": "^3.2.10",
-    "@testing-library/jest-dom": "^4.2.4",
```

Comment:
These deps were giving me ```npm install``` troubles, but aren't actually used. Removing for now. We can re-add jest + enzyme if / when we add proper unit tests for our react app.